### ISSUE TITLE
Kernel fails to start with 8.2.8 global developer ROM of MIUI.

### ISSUE THREAD

**Issue by MrBrianGale** [JonasCardoso/android_kernel_xiaomi_msm8996#95]
[console-ramoops.txt](https://github.com/JonasCardoso/android_kernel_xiaomi_msm8996/files/1729143/console-ramoops.txt)
Attached is my console-ramoops file which from my understanding is the replacement to the last_kmsg file in newer android versions.
This seems to be happening on global developer version 8.2.8.  Some of the data looks a bit screwy and I am not sure why... like "size 3% MiB" or U3ing instead of using.
Although, looking at the log, I am not positive if this is the TWRP booting log or the log that is being unhappy booting the Floppy Kernel.

**Comment by JonasCardoso** [comment]
I noticed this problems, new firmware does not work

**Comment by MrBrianGale** [comment]
doh :( oh well.  Guess I'll just wait and hope that Xiaomi fixes their firmware or that a new version of floppy comes out with the firmware fixes.  Although I expect the new floppy version likely won't support it until the global stable bumps up the firmware, eh?

**Comment by kartun83** [comment]
Lineage 15.1 from 10 Feb already requires 8.2.8, so it would be nice to have floppy compatible with it

**Comment by kkwpsv** [comment]
I meet the same issue.......I hope you can fix it : )

**Comment by MrBrianGale** [comment]
I was reading that rolling back the firmware (flashing the old firmware after flashing the kernel) may help resolve things - https://forum.xda-developers.com/mi-5s-plus/development/mi5s-plus-floppy-kernel-t3723681/page15
Not sure if it fixes things or makes stuff worse as I have not tried it.

**Comment by MrBrianGale** [comment]
Update.  I tried flashing the ROM->magisk->kernel->old firmware->magisk and the phone doesn't boot :(  This method was used because I installed the ROM ages back and installed magisk.  Going to attempt ROM->kernel->old firmware and see what happens (temporarily skipping magisk in case that is causing the problems).  Likely won't get to this today though.
never mind... got this done today and it seems a lot happier.  Phone booted up and have the Floppy Kernel on there.  So the trick is:
ROM->Kernel->Firmware->Magisk
I used firmware 8.1.11 and it seems to work with this kernel and with the global developer ROM.
